Formula/rbspy.rb

```diff
@@ -0,0 +1,18 @@
+class Rbspy < Formula
+  desc "Sampling profiler for Ruby"
+  homepage "https://github.com/rbspy/rbspy"
+  url "https://github.com/rbspy/rbspy/archive/v0.1.8.tar.gz"
+  sha256 "87f759acd9d660178737b9f24cc07f0113a81a9fffc1604bae2c756d7f4d815b"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build"
+    bin.install "target/debug/rbspy"
+  end
+
+  test do
+    output = shell_output("#{bin}/rbspy -V")
```

Comment:
I know this is frowned upon however `rbspy` requires `sudo` on mac to perform most of the operations which I something I don't think we want to introduce into a CI test so I settled for the next best thing. I have raised rbspy/rbspy#88 which might help us get a non-root bit of output here.